modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -16,7 +16,7 @@ const calcCols = (colors: number): number =>
 const calcColsOption = (editor: Editor, numColors: number): number => {
   const calculatedCols = calcCols(numColors);
   const fallbackCols = option('color_cols')(editor);
-  return defaultCols === calculatedCols ? fallbackCols : calculatedCols;
+  return defaultCols === calculatedCols || fallbackCols !== defaultCols ? fallbackCols : calculatedCols;
```

Comment:
Does `defaultCols === calculatedCols` ever make sense? I'm not sure it does 🤔 

Given that `color_cols` has a sensible default, maybe we don't need `defaultCols` at all.
https://github.com/tinymce/tinymce/blob/00ecd8ba31446c16c10d4195db6da9d054e221b0/modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts#L98

Try commenting it out, always use the option value, and see if any tests fail 😂